ui/src/VoxettaApp.js

```diff
@@ -60,28 +102,102 @@ export class VoxettaApp extends LitElement {
      */
     displayComponents() {
         switch (this.state) {
-            case States.RECORD_PAGE:
+            case States.ACTIVE_RECORD_PAGE:
+                return html`
+                    ${this.getActiveRecordTemplate()}
+                `;
+            case States.INACTIVE_RECORD_PAGE:
                 return html`
-                    <vox-user-icon 
-                        @enter-form="${() => { this.state = States.USER_FORM }}">
-                    </vox-user-icon>
-                    <vox-prompts></vox-prompts>
-                    <vox-record-button></vox-record-button>
+                    ${this.getInactiveRecordTemplate()}
                 `;
             case States.USER_FORM:
-                 return html`
-                    <vox-user-form
-                        .userId = ${this.userId}
-                        .gender = ${this.gender}
-                        .userAge = ${this.userAge}
-                        .deviceType = ${this.deviceType}
-                        @update-user-info="${(e) => { 
-                            this.updateUserInformation(e.detail.userInfo);
-                            this.cookieService.makeUserInfoCookie(e.detail.userInfo); }}"
-                        @exit-form="${() => { this.state = States.RECORD_PAGE }}">
-                    </vox-user-form>
+                return html`
+                    ${this.getUserFormTemplate()}
                 `;
-        }     
+        }      
+    }
+
+    /**
+     * The componenets associated with the active recording state. 
+     * @returns {HTML} The HTML template for the active recording state.
+     */
+    getActiveRecordTemplate() {
+        return html`
+            <div class="header">
+                <vox-user-icon 
+                    @enter-form="${() => {this.state = States.USER_FORM}}">
+                </vox-user-icon>
+            </div>
+            <div class="prompts">
+                <vox-prompts
+                    .promptState=${this.promptState}
```

Comment:
Remove all references to this.promptState in this file

Reply:
my bad! removed